libclambcc/ClamBCExtendPHIsTo64Bit/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2021 Cisco Systems, Inc. and/or its affiliates. All rights reserved.
```

Comment:
Did you mean to add this cmake file for this PR?

Reply:
It should have already been there.  I force-pushed it to my branch, and forgot that that hadn't been merged upstream.  I really don't care, I am going to clean up the commit history when this is ready to merge into main, anyway, unless you would rather I did it a different way?